### PR TITLE
#167727739 Avail requests for approval

### PR DESCRIPTION
**Pull request by Mcdavid95**
#### What does this PR do?
- Makes available pending approvals for the assigned manager

#### Description of tasks to be done
- add controller to get requests assigned to a manager
- add route/endpoint to get requests assigned to a manager

#### How to manually test
- Clone the repo
- Run `npm install`
- Create database using `createdb {database_name}`
- create `.env` file and add database url under the key `DATABASE_URL_TEST`
- run `npm run seed`
- Using postman or any http request tool, make a get request to `/api/v1/requests/manager/:userId` to get user details
- You can optionally pass in  `page` and `perPage` to the request as query string as page number and number of items per page respectively to paginate the request
- Navigate to `/api/v1/docs` to test further with swagger ui 

#### Relevant Pivotal tracker stories
- [#167727739](https://www.pivotaltracker.com/story/show/167727739)

### CODE REVIEW

File: src/controllers/requestController.js [hunk 1]
```diff
@@ -119,9 +119,42 @@ const searchRequest = async (req, res) => {
...
 };
+/**
+ * @function getManagerRequest
+ * @param {Object} req - server request
+ * @param {Object} res - server response
+ * @returns {Object} - custom response
+*/
+const getManagerRequest = async (req, res) => {
+  try {
+    const { params: { userId }, decoded, query: { page, perPage } } = req;
```
Comment (Mcdavid95) [inline src/controllers/requestController.js]: Add ternary  conditional to check if person, is a super admin

Comment (Mcdavid95) [inline src/controllers/requestController.js]: make userId optional

**Comment by tejiri4** [comment]
Rename your PR title and rebase

**Review by tejiri4 — APPROVED** [review]
LGTM

**Pull request merged**